syntax = "proto3";
package workspace;





/*
Init请求
*/
message InitRequest {
    /*
    sdk限制字段使用
    */
    repeated string XXX_RestFieldMask = 1 ;
    /*
    程序包Id
    */
    string packageId = 2 ;
    /*
    版本Id
    */
    string versionId = 3 ;
}

/*
Init返回
*/
message InitResponse {
    /*
    返回状态码
    */
    int32 code = 1 ;
    /*
    错误信息
    */
    string error = 2 ;
    /*
    返回消息
    */
    string message = 3 ;
    /*
    版本Id
    */
    string data = 4 ;
}

/*
InitApi返回
*/
message InitResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    workspace.InitResponse data = 4 ;
}

